### ISSUE TITLE
Action state frame index

### ISSUE THREAD

**Issue by Walnut356** [hohav/peppi#55]
melee isn't consistent about which frame is considered the "first" frame of an action state. Knowing that you're on the first frame of an action can be pretty useful when generating stats, so it might be worth adding this data somewhere.

[Altf4 already has a CSV](https://github.com/altf4/libmelee/blob/master/melee/actiondata.csv#L3) containing all the info necessary, it'd just be a matter of representing it internally and implementing a convenient way to check. In libmelee he reindexes all action states, but i don't think that's necessary at the base level (though may be convenient with higher levels of abstraction like stats generation)

**Comment by NickCondron** [comment]
I agree this is useful info to have. I was thinking along similar lines when I made this PR https://github.com/hohav/peppi/pull/13 that added other info about characters. I closed it because it makes more sense for this kind of data to live outside of peppi. There aren't really any repositories where this exists though. I think short term, if there are stats that rely on these values, then keep them in a module related to stats.

This is the closest thing I've found: https://github.com/pfirsich/meleeDat2Json
